src/Symfony/Bundle/FrameworkBundle/Resources/config/http_client.php

```diff
@@ -30,6 +30,7 @@
             ])
             ->call('setLogger', [service('logger')->ignoreOnInvalid()])
             ->tag('monolog.logger', ['channel' => 'http_client'])
+            ->tag('kernel.reset', ['method' => 'reset', 'on_invalid' => 'ignore'])
```

Comment:
What could possibly be other values for `on_ignore`? What about `ignore_on_invalid => true`?

I guess it's to reuse the DI `on-invalid="null/ignore"` known terms, but here having just a key is one less thing to think/search.

Reply:
I don't know which other values are possible, but tag attributes allow only string, so `true` is not possible, we can only approximate it.

Reply:
I didn't remember such a limitation 🤔  
Tags are even phpized by the XmlFileLoader.